katas/content/nonlocal_games/index.md

```diff
@@ -53,9 +51,147 @@ To start with, let's take a look at how you would play the classical variant of
+    0 & 0 & 0.9239 & 0.3827\\
+    0 & 0 & -0.3827 & 0.9239
+\end{bmatrix}$$
+This performs the $R_y$ rotation by $\pi/8$ radians clockwise on Bob's qubit while leaving Alice's qubit unchanged.
+
+- If Bob's bit were 1, he would rotate his qubit counterclockwise by $\pi/8$, applying a very similar operator
+
+$$\begin{bmatrix}
+    0.9239 & -0.3827 & 0 & 0\\
+    0.3827 & 0.9239 & 0 & 0\\
+    0 & 0 & 0.9239 & -0.3827\\
+    0 & 0 & 0.3827 & 0.9239
+\end{bmatrix}$$
+
+Therefore, when Alice has bit 0, the application of the rotation operator to the Bell state gives
+$$\begin{bmatrix}
+    0.6533 \\
+    -0.2706 \\
+    0.2706 \\
+    0.6533
+\end{bmatrix} \text{ or }
+\begin{bmatrix}
+    0.6533\\
+    0.2706\\
+    -0.2706\\
+    0.6533
+\end{bmatrix}$$
+depending on whether Bob holds 0 (left-hand case) or 1 (right-hand case).
+
+The result of AND on their input bits will always be 0; thus they win when their outputs agree.  These two cases correspond to the top and bottom elements of the vectors above, with a combined probability of $(0.6533)^2 + (0.6533)^2 = 0.4268 + 0.4268 = 0.8536$, so they have an 85.36% win chance.
+
+## Case 2: Alice holds bit 1
+
+When Alice holds bit 1, she measures in basis X (or, equivalently, Hadamard-transforms her qubit, leaving Bob's be, before making her Z-basis measurement).  This corresponds to applying the operator
+$$\begin{bmatrix}
+    0.7071 & 0 & 0.7071 & 0\\
+    0 & 0.7071 & 0 & 0.7071\\
+    0.7071 & 0 & -0.7071 & 0\\
+    0 & 0.7071 & 0 & -0.7071
+\end{bmatrix}$$
+to the Bell state, resulting in a vector of:
+$$\begin{bmatrix}
+    0.5\\
+    0.5\\
+    0.5\\
+    -0.5
+\end{bmatrix}$$
+
+Now, one of the two rotation operators is applied depending on what bit Bob holds, transforming this vector into:
+$$\begin{bmatrix}
+    0.6533 \\
+    0.2706 \\
+    0.2706 \\
+    -0.6533
+\end{bmatrix} \text{ or }
+\begin{bmatrix}
+    0.2706\\
+    0.6533\\
+    0.6533\\
+    -0.2706
+    \end{bmatrix}$$
+
+When Bob holds 0, they still want to return the same parity, which they again do with 85.36% probability (left-hand vector above).
+But when Bob holds 1, the AND condition is now true and the players want to answer in opposite parity. This corresponds to the second and third elements of the right-hand vector above.
+Thanks to the "magic" of the combination of the counterclockwise rotation and Hadamard transform, they now do this with probability $(0.6533)^2 + (0.6533)^2 = 0.8536$ and thus 85.36% becomes their win odds once more.
+
+## Side notes
+
+- If Bob never rotated his qubit, their entangled state would remain the Bell state if Alice held bit 0 and the state corresponding to $\frac12 \big(|00\rangle + |01\rangle + |10\rangle - |11\rangle\big)$ if Alice held bit 1.
+
+While she and Bob would have a 100% success probability against Alice's 0 bit, they would have only a 50% chance of success if she held bit 1, and thus their win chance would revert to the 75% of the classical strategy again.
```

Comment:
This needs to be part of the first bullet point; we need to either remove the empty line or indent the line after it

Reply:
The sentence is merge with first bullet point (empty line is removed).
Screenshot is attached.
![index_paragraph_merge](https://github.com/user-attachments/assets/9ba3e7f9-aae0-44c2-a5f4-ce4a155e5da4)